src/commands/ai/text-gen.js

```diff
@@ -0,0 +1,86 @@
+'use strict';
+
+const BoxCommand = require('../../box-command');
+const { flags } = require('@oclif/command');
+const utils = require('../../util');
+
+class AiTextGenCommand extends BoxCommand {
+	async run() {
+		const { flags, args } = this.parse(AiTextGenCommand);
+		let options = {};
+
+        if (flags['dialogue-history']) {
+            options.dialogueHistory = flags['dialogue-history'];
+        }
+        options.prompt = flags.prompt;
+        options.items = flags.items;
+        let answer = await this.client.ai.textGen({
+            prompt: options.prompt,
+            items: options.items,
+            dialogue_history: options.dialogueHistory,
+        });
+
+        await this.output(answer);
+	}
+}
+
+AiTextGenCommand.description = 'Sends an AI request to supported LLMs and returns an answer specifically focused on the creation of new text.';
+AiTextGenCommand.examples = ['box ai:text-gen --dialogue_history=prompt="What is the status of this document?",answer="It is in review",created-at="2024-07-09T11:29:46.835Z" --items=id=12345,type=file --prompt="What is the status of this document?"'];
+AiTextGenCommand._endpoint = 'post_ai_text_gen';
+
+AiTextGenCommand.flags = {
+	...BoxCommand.flags,
+
+    'dialogue-history': flags.string({
+        required: false,
+        description: 'The history of prompts and answers previously passed to the LLM.',
+        multiple: true,
+        parse(input) {
+            const record = {};
+            const obj = utils.parseStringToObject(input);
```

Comment:
When I run the following command
```
./bin/run ai:text-gen --dialogue-history=prompt="What is the status of this document?",answer="It, is in review",created-at="2024-07-29T04:54:04.811-07:00" --items=id=1453000382056,type=file --prompt="What is the status of this document?";
```

where within an `answer` property I put a text with comma  -> `It, is in review`,
then the following request is sent:

```
{
  "dialogueHistory": [
    {
      "prompt": "What is the status of this document?",
      "answer": "It",
      "created_at": "2024-07-29T11:54:04+00:00"
    }
  ],
  "prompt": "What is the status of this document?",
  "items": [
    {
      "id": "1453000382056",
      "type": "file"
    }
  ]
}
```

As you can see, everything the starting from the comma `,` character in `answer` property was cut.
Can you take a look into that?